src/server/service/kv.rs

```diff
@@ -125,6 +125,7 @@ impl<T: RaftStoreRouter<RocksEngine> + 'static, E: Engine, L: LockManager> Servi
         }
     }
 
+    #[allow(dead_code)]
```

Comment:
Why dead code?

Reply:
Do you mean if it's real dead code, we can remove it?